src/datasources/tag/components/TagQueryEditor.tsx

```diff
@@ -59,4 +67,6 @@ const tooltips = {
 
   workspace: `The workspace to search for the given tag path. If left blank, the plugin
               finds the most recently updated tag in any workspace.`,
+
+  properties: `If toggled, include the tag's properties in the query result.`,
```

Comment:
Please review this string @MelissaHilliard

Reply:
```suggestion
  properties: `If enabled, include tag properties in the query result.`,
```